src/kitty.rs

```diff
@@ -24,12 +26,6 @@ fn setup(mut commands: Commands) {
 #[derive(Resource)]
 pub struct KittyEnabled;
 
-impl Drop for KittyEnabled {
```

Comment:
I wonder if there's a way to (deterministically) get this to occur before the other things that are dropped.

Perhaps by calling remove_resource in response to AppExit? Does doing that allow us to order the systems perhaps?

I think we might be able to handle this kind of the same way that a panic hook in normal ratatui does - treat the drop as the last ditch effort to cleanup, but to think of the app actively cleaning up when it can.